CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# Release 2.35.0 22-03-2022
+
+- Update Envoy to v1.21.1 and Bazel to v5.0.0 (#670)
```

Comment:
NIT: only "- Update Envoy to v1.21.1 (#670)" for line 3

Reply:
Done.